src/api/contributionsApi.testData.ts

```diff
@@ -0,0 +1,71 @@
+import { EpicTests } from '../lib/variants';
+
+export const configuredTests: EpicTests = {
+    tests: [
+        {
+            name: '2020-02-11_enviro_fossil_fuel_r2_Epic__no_article_count',
+            isOn: true,
+            locations: [],
+            tagIds: [
+                'environment/series/the-polluters',
+                'environment/environment',
+                'environment/the-forgotten-climate-change-crises',
+                'environment/drought',
+                'environment/endangered-habitats',
+                'environment/endangeredspecies',
+                'environment/climate-change',
+                'environment/activism',
+                'environment/climate-change',
+                'science/scienceofclimatechange',
+                'environment/renewableenergy',
+                'environment/renewableenergy',
```

Comment:
Duplicated tagId?

Reply:
Ha good spot! (In my defence, this is copied from some real config which also has the duplicate 😂)